examples/life-expectancy/le_utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2021 VMware, Inc.
```

Comment:
Should not le_utils folder be in the data job directory (live-expectancy/live-expectancy) ?

Reply:
I didn't put it in the data job directory because I assumed that all the files contained in the data job directory are processed by vdk, and I don't want le_utils being processed by vdk. However, if you want, I can move it within the data job dir.

Reply:
Perfect! I'm going to update the code!

Reply:
Just updated the code and removed

`current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
parent_dir = os.path.dirname(current_dir)
sys.path.insert(0, parent_dir)`